integration/mediation-tests/tests-platform/tests-userstore/src/test/resources/security/policies/scenario1-policy.xml

```diff
@@ -0,0 +1,58 @@
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's update the license headers with LLC. Please refactor other places as well.